common/common.go

```diff
@@ -419,10 +424,10 @@ func printMessage(printResult bool, format string, args ...interface{}) {
 
 // ExecuteBatchRun starts batch run(s) and wait for its completion with showing progress
 func ExecuteBatchRun(urlBase string, apiToken string, organization string, project string,
-	httpHeadersMap map[string]string, testSettingsNumber int, setting string,
+	httpHeadersMap map[string]string, testSettingsNumber int, branchName string,  setting string,
```

Comment:
[nits]
```suggestion
	httpHeadersMap map[string]string, testSettingsNumber int, branchName string, setting string,
```

Reply:
Thank you
Fixed in [8b56016](https://github.com/Magic-Pod/magicpod-api-client/pull/28/commits/8b560168d40c9773fda14060fe4b730c167f7b24)